src/main/webapp/repo-bootstrap/global/blueprints/1000_website_editorial/config/studio/ui.xml

```diff
@@ -109,9 +109,10 @@
                                 <configuration>
                                     <title id="GlobalMenu.EncryptionToolEntryLabel" defaultMessage="Encryption Tool"/>
                                     <icon id="@material-ui/icons/LockOutlined" />
-                                    <widget id="craftercms.components.LegacySiteToolsFrame">
+                                    <widget id="craftercms.components.SiteEncryptTool">
                                         <configuration>
-                                            <tool>encrypt-tool</tool>
+                                            <logType>preview</logType>
```

Comment:
The only component that should be getting this prop is the `LogConsole` (`craftercms.components.LogConsole`)